lib/datadog/opentelemetry/sdk/span_processor.rb

```diff
@@ -87,6 +89,21 @@ def start_datadog_span(span)
           datadog_span.set_error([nil, span.status.description]) unless span.status.ok?
           datadog_span.set_tags(span.attributes)
 
+          unless span.links.nil?
+            datadog_span.links = span.links.map do |link|
+              Datadog::Tracing::SpanLink.new(
+                Datadog::Tracing::TraceDigest.new(
+                  trace_id: link.span_context.trace_id.unpack1('Q'),
```

Comment:
```suggestion
                  trace_id: to_datadog_id(link.span_context.trace_id),
```

Can you wrap `unpack1('Q')` into a method, so that we know what it does in the future?

For example, this is how we do the inverse operation: https://github.com/DataDog/dd-trace-rb/blob/53953acaed8fc227c9b0e748d1e47a65e7e11954/lib/datadog/opentelemetry/sdk/propagator.rb#L84-L88